packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -51,9 +51,9 @@ export class DropdownBase extends BaseComponent<IDropdownInternalProps, IDropdow
     options: [] as any[]
   };
 
-  private _host = createRef<HTMLDivElement>();
+  private _host = React.createRef<HTMLDivElement>();
   private _focusZone = createRef<IFocusZone>();
```

Comment:
Why not convert this createRef as well?

Reply:
Right; remove the import and you'll have to fix em :)

If there's a good reason to keep, we need to discuss why (and add lots of comments) but I think this is just an oversight.

Reply:
maybe i can just do it here.

Reply:
fixed.